File: 101/exercise_rework/easy_9/welcome_stranger.rb
# welcome_stranger.rb

def greetings(arr, hsh)
  first_name = arr[0]
  middle_initial = arr[1]
  last_name = arr[2]
  title = hsh[:title]
  occupation = hsh[:occupation]

  "Hello, #{first_name} #{middle_initial} #{last_name}! Nice to have" \
  " a #{title} #{occupation} around."
end

File: 210/lesson_5/practice_problems1/welcome_stranger.js
// welcome_stranger.js

function greetings(arr, obj) {
  var msg = `Hello, ${arr.join(' ')}! Nice to have a ${obj.title} ${obj.occupation} around.`;

  console.log(msg);
}

greetings(['John', 'Q', 'Doe'], { title: 'Master', occupation: 'Plumber' });

// console output
//=> Hello, John Q Doe! Nice to have a Master Plumber around.